System/Vector.h

```diff
@@ -440,7 +418,7 @@ namespace RTE {
 		/// <returns>The resulting Vector.</returns>
 		Vector operator/(const float &rhs) const {
 			Vector returnVector(0, 0);
-			if (rhs) { returnVector.SetXY(m_X / rhs, m_Y / rhs); }
+			if (rhs != 0) { returnVector.SetXY(m_X / rhs, m_Y / rhs); }
```

Comment:
😮 
Also, why construct an empty Vector here? Why not to it inline like `*`?

Reply:
Mainly I did it because there's also an if-statement here, but I can condense it down to a ternary.